zap/src/main/java/org/parosproxy/paros/Constant.java

```diff
@@ -172,6 +173,7 @@
 import org.zaproxy.zap.control.AddOnLoader;
 import org.zaproxy.zap.control.ControlOverrides;
 import org.zaproxy.zap.extension.autoupdate.OptionsParamCheckForUpdates;
+import org.zaproxy.zap.extension.pscan.scanner.RegexAutoTagScanner.TYPE;
```

Comment:
Hardcode the value, the pscan classes will be deprecated when the add-on is moved.